Add one entry on releasing resources (#191)
EGI-Foundation/documentation: content/en/users/cloud-compute/faq/_index.md

If you don\'t have public IPs for all the VMs to be managed, you can also use
one as a gateway as described in the
[Ansible FAQ](https://docs.ansible.com/ansible/latest/reference_appendices/faq.html#how-do-i-configure-a-jump-host-to-access-servers-that-i-have-no-direct-access-to).

### How can I release resources without destroying my data?

Whenever you delete a VM, the ephemeral disks associated with it will be also
deleted so if you don't plan to use your VM for some time but still be able
to recover the data or boot your VM in the same state, you will need to use

> some of the following strategies:

Reviewer: (not proposing as suggestion so that you can format properly the line length)

```
> Whenever you delete a VM, the ephemeral disks associated with it will also be deleted.

If you don't plan to use your VM for some time but still want to be able
to recover the data or boot your VM in the same state, you will need to use
some of the following strategies:
```

Author: I rephrased a bit

- Use a volume to store the data to be kept: Check the [Storage section of the
  documentation](../storage) to learn how to use volumes. If you start your VM
  from a volume, the VM can be destroyed and recreated easily. OpenStack

>   docs cover how to [start a VM from a volume with CLI](https://docs.openstack.org/nova/latest/user/launch-instance-from-volume.html)

Reviewer: ```suggestion
  documentation cover how to [start a VM from a volume with CLI](https://docs.openstack.org/nova/latest/user/launch-instance-from-volume.html)
```

  or [using the Horizon dashboard](https://docs.openstack.org/horizon/latest/user/launch-instances.html)

- Suspend or shelve instances: Suspending a VM will release CPU and memory

>   usage of a VM but keep a copy of the RAM contents so it can be restored

Reviewer: Confusing: it release RAM content that is lost but you can restore at the exact same state

> - Suspend or shelve instances: Suspending a VM will pause a VM, releasing CPU

Reviewer: ```suggestion
- Suspend or shelve instance: Suspending a VM will pause a VM, releasing CPU
```

  and memory, and allowing to resume later in time at the exact same state.
  Shelving shuts down the VM, thus RAM contents will be lost but disk will be
  kept. This releases more resources from the provider while still allows to
  easily boot the VM back without losing disk contents.

> - Create snapshot of instances: a snapshot will create a new VM image at your

Reviewer: ```suggestion
- Create snapshot of instance: a snapshot will create a new VM image at your
```

Reviewer: I understand that either it should have been snapshots of instances or have instance singular, thus suggested to just use instance.

> - Create snapshot of instances: a snapshot will create a new image on your
>   provider that can be used to boot a new instance of the VM with the same
>   disk contents. You can use this technique for creating a base template
>   image that can be later re-used to start similar VMs easily.

Reviewer: ```suggestion
- Create snapshot of instances: a snapshot will create a new VM image at your
  provider that can be used to boot a new instance of the VM with the same
  disk content. You can use this technique for creating a base template
  image that can be later re-used to start similar VMs easily.
```
